### PR TITLE
Review Brand & Wishlist Controllers

### PR DESCRIPTION
**Pull request by arijit1111**
Review Brand & Wishlist Controllers

### CODE REVIEW

**Comment by properweb** [comment]
Task linked: [CU-2yehr3e Customer Wishlist](https://app.clickup.com/t/2yehr3e)

File: Modules/Backend/Http/Controllers/VendorController.php [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                'featured_image' => $items->featured_image,
+                'profile_photo' => $items->profile_photo,
+                'cover_image' => $items->cover_image,
+                'logo_image' => $items->logo_image
+
+            );
+            $count++;
+        }
+        echo $json_data = json_encode(array("data" => $data));
+        exit;
```
Comment (shoaibrehman) [inline Modules/Backend/Http/Controllers/VendorController.php:89]: what's the purpose of this echo? And, then the exit? 

If function is not required for execution then please remote it.

File: Modules/Order/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -14,11 +13,15 @@
...
+use Modules\Wordpress\Http\Controllers\WordpressController;
+use Modules\Shopify\Http\Controllers\ShopifyController;
+use Modules\Product\Entities\BrandStore;
 use DB;
 
 class OrderController extends Controller
 {
 
+
     public function store(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:28]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 3]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
                     'order_det' => $order,
                     'cart_det' => $cart_arr,
                 );
+                $response = ['res' => true, 'msg' => 'Your product successfully placed in order', 'data' => $data];
+            }
+            return response()->json($response);
+        }
+    }
 
+    public function index(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:145]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 4]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+            $response = ['res' => true, 'msg' => "", 'data' => $data];
+            return response()->json($response);
+        } else {
+            $response = ['res' => false, 'msg' => "No record found", 'data' => ""];
+            return response()->json($response);
+        }
+    }
 
-                $response = ['res' => true, 'msg' => 'Your product successfully placed in order', 'data' => $data];
+    public function show(Request $request, $orderNumber)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:204]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 5]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+                'total_price' => $total_price,
+                'related_orders' => $related_orders
+            );
+        }
+
+        $response = ['res' => true, 'msg' => "", 'data' => $data];
+        return response()->json($response);
+    }
+
+    public function packingSlip(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:292]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 6]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+                    );
+                }
+            }
+        }
+
+        $response = ['res' => true, 'msg' => "", 'data' => $data];
+        return response()->json($response);
+    }
+
+    public function accept(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:353]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 7]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+
+            $data = [];
+
+
+            $response = ['res' => true, 'msg' => '', 'data' => $data];
+        }
+        return response()->json($response);
+    }
+
+    public function changeDate(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:401]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 8]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+                }
+            }
         }
+
+
+        $response = ['res' => true, 'msg' => "", 'data' => $data];
+        return response()->json($response);
     }
 
+    public function changeAddress(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:431]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 9]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+            $msg .= "<strong>Reason for cancelling</strong><br>";
+            $msg .= $request->cancel_reason_title . "<br>";
+            $msg .= $request->cancel_reason_desc . "<br>";
+            $data = array('email' => $retailer_user->email, 'order_number' => $order->order_number);
+        }
+        $response = ['res' => true, 'msg' => "", 'data' => ""];
+        return response()->json($response);
+    }
+
+    public function split(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:492]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 10]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+            $order->sub_total = Cart::where('user_id', $order->user_id)->where('brand_id', $order->brand_id)->where('order_id', $order->id)->sum('amount');
+            $order->quantity = Cart::where('user_id', $order->user_id)->where('brand_id', $order->brand_id)->where('order_id', $order->id)->sum('quantity');
+            $order->total_amount = $order->sub_total;
+            $order->save();
+        }
+        $response = ['res' => true, 'msg' => "", 'data' => ""];
+        return response()->json($response);
+    }
+
+    public function csv(Request $request)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:574]: phpdocs is still missing

File: Modules/Order/Http/Controllers/OrderController.php [hunk 11]
```diff
@@ -123,19 +125,493 @@ public function store(Request $request)
...
+                $data .= $orderDetails->order_number . "," . $item->product_sku . "," . $item->product_name . "," . $item->quantity . "\n";
+            }
+        }
+        header('Content-Type: application/csv');
+        header('Content-Disposition: attachment; filename="' . $filename . '.csv"');
+        echo $data;
+        exit();
+    }
+
+    private function syncExternal($prdct_arr, $brandId)
```
Comment (shoaibrehman) [inline Modules/Order/Http/Controllers/OrderController.php:601]: phpdocs is still missing

**Review by shoaibrehman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**